tox.ini

```
@@ -33,17 +32,13 @@ passenv =
     SQLALCHEMY_UTILS_TEST_*
 recreate = True
 
-[testenv:flake8]
+[testenv:ruff]
 skip_install = True
 recreate = False
-deps = pre-commit
-commands = pre-commit run --hook-stage manual --all flake8
-
-[testenv:isort]
-skip_install = True
-recreate = False
-deps = pre-commit
-commands = pre-commit run --hook-stage manual --all isort-check
+deps = ruff>=0.1.0
+commands =
+    ruff check .
+    ruff format --check .
```

To keep the ruff version in sync with the pre-commit-enforced version, this should keep the pre-commit-based invocation style:

```suggestion
deps = pre-commit
commands =
    pre-commit run --all ruff
    pre-commit run --all ruff-format
```

This will eliminate discrepancies between running a newer ruff version picked up by the test suite, and actually committing changes.